test/TestUtils/TestProjectFixture.cs

```diff
@@ -0,0 +1,290 @@
+﻿using Microsoft.DotNet.Cli.Build;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.DotNet.CoreSetup.Test
+{
+    public class TestProjectFixture
```

Comment:
Livar may kill me, but I'll add one :smile:

---
In reply to: [66352494](https://github.com/dotnet/core-setup/pull/97#discussion_r66352494) [](ancestors = 66352494,66350938,66350076,66348239,66107984,66102620)